schemas/objects/projectPrefinancing.js

```diff
@@ -75,7 +75,7 @@ export default {
           type: 'text',
         },
       ],
-      validation,
+      validation: Rule => validation(Rule).max(6),
```

Comment:
this is part of a commit I hadn't pushed before merging sanity prefinancing work https://github.com/regen-network/regen-sanity/pull/56